lib/PlaidClient.js

```diff
@@ -61,14 +61,19 @@ var requestWithAccessToken = function(path) {
   };
 };
 
-var requestWithIncomeVerificationId = function(path) {
-  return function(incomeVerificationId, options, cb) {
-    return this._authenticatedRequest({
-      path: path,
-      body: {
-        income_verification_id: incomeVerificationId,
-      }
-    }, cb);
+var requestWithIncomeVerificationId = function (path, requestOptions = {}) {
```

Comment:
Not sure if this is the past way to keep this extensible and pass the options to `authenticatedRequest`. I didn't wanted to shadow `options` either. 

Let me know what to do here.